src/components/date-range-input-controller.vue

```diff
@@ -139,54 +160,77 @@ export default {
       type: Boolean,
       default: false
     },
+    withFullScreenPortal: {
+      type: Boolean,
+      default: false
+    },
     minimumNights: {
       type: Number,
       default: 1
     },
     isOutsideRange: {
       type: Function,
-      default: function() {}
+      default: function(day) {
+        return !isInclusivelyAfterDay(day, moment())
+      }
     },
     displayFormat: {
       type: Function,
       default: function() {
-        return moment.localeData().longDateFormat("L");
+        return moment.localeData().longDateFormat('L')
       }
     },
-    handleFocusChange: {
+
+    onFocusChange: {
       type: Function,
       default: function() {}
     },
-    handleClose: {
+    onClose: {
       type: Function,
       default: function() {}
     },
-    handleDatesChange: {
+    onDatesChange: {
       type: Function,
       default: function() {}
     },
-    handleKeyDownArrowDown: {
+    onKeyDownArrowDown: {
       type: Function,
       default: function() {}
     },
-    handleKeyDownQuestionMark: {
+    onKeyDownQuestionMark: {
       type: Function,
       default: function() {}
     },
-    isRTL: {
-      type: Boolean,
-      default: false
+
+    customInputIcon: {
+      type: String,
+      default: null
     },
+    customArrowIcon: {
+      type: String,
+      default: null
+    },
+    customCloseIcon: {
+      type: String,
+      default: null
+    },
+
+    // accessibility
     isFocused: {
       type: Boolean,
       default: false
     },
+
+    // i18n
     phrases: {
       type: Object,
-      default: function() {
```

Comment:
https://vuejs.org/v2/guide/components-props.html#Prop-Validation
Object or array defaults must be returned from a factory function